addons/account/models/account_tax.py

```diff
@@ -1605,25 +1606,54 @@ def _round_base_lines_tax_details(self, base_lines, company, tax_lines=None):
             if not tax_amounts['base_lines']:
                 continue
 
-            base_line = tax_amounts['reference_base_line'] = max(
-                tax_amounts['base_lines'],
-                key=lambda base_line: base_line['tax_details']['total_included_currency'],
+            tax_amounts['sorted_base_line_x_tax_data'] = [
+                (
+                    base_line,
+                    next(
+                        (
+                            tax_data
+                            for tax_data in base_line['tax_details']['taxes_data']
+                            if tax_data['tax'] == tax and tax_data['is_reverse_charge'] == is_reverse_charge
+                        ),
+                        None,
+                    )
+                )
+                for base_line in sorted(
+                    tax_amounts['base_lines'],
+                    key=lambda base_line: -base_line['tax_details']['total_included_currency'],
```

Comment:
```suggestion
                    key=lambda base_line: -abs(base_line['tax_details']['total_included_currency']),
```
so that the lines with the biggest absolute total come first

Reply:
@antoine162 I want the negative lines to be considered at the very end so no abs here